parser/parser.go

```
@@ -109,14 +119,14 @@ func parse(path string) {
 		panic(err)
 	}
 
-	// Dictionary to hold the row data, keyed by row header
-	rowInfo := getRowInfo(doc)
-	// Dictionary to hold the class info, keyed by data label
-	classInfo := getClassInfo(doc)
-
-	// Get the class and course num by splitting classInfo value
+	//recover from failing to parse a section
+	defer func() {
+		if r := recover(); r != nil {
+			utils.VPrintf("Recovered from panic, skipping section %s: %v", path, r)
+		}
+	}()
+	parseSection(getRowInfo(doc), getClassInfo(doc))
```

I think we want to avoid recovering from a panic here -- generally, failing to parse a section is a panic scenario rather than something that should be skipped.

This will also cause a silent failure unless we're parsing in verbose mode, which is something we want to avoid